packages/security_detection_engine/changelog.yml

```
@@ -1,5 +1,10 @@
 # newer versions go on top
 # NOTE: please use pre-release versions (e.g. -dev.0) until a package is ready for production
+- changes:
+    - description: Release security rules update
+      link: https://github.com/elastic/integrations/pull/5238
+      type: enhancement
+  version: 8.4.3-beta.1
```

Shouldn't this look something like: ?

```suggestion
-   version: 8.4.3-beta.1
    changes:
    - description: Release security rules update
      link: https://github.com/elastic/integrations/pull/5238
      type: enhancement

```

---

Yes it should be -> https://github.com/elastic/detection-rules/issues/2351